src/types.ts

```diff
@@ -27,7 +28,7 @@ export type CioAutocompleteProps = CioClientConfig & {
   openOnFocus?: boolean;
   onSubmit: OnSubmit;
   onFocus?: () => void;
-  onChange?: (string) => void;
+  onChange?: (arg0: string) => void;
```

Comment:
Can we make input or value instead of arg0

Reply:
👍 The typescript linter suggested making it arg0: string so I followed it without thinking 😂